### ISSUE TITLE
[jss-expand] Observables not respected properly

### ISSUE THREAD

**Issue by brianmhunt** [cssinjs/jss#742]
Using TC39-compliant observables from Knockout:

```js
  const observable = ko.observable('normal')
  animation: {
         name: 'anim',
         duration: '0.25s',
         direction: observable,
  }
```

One would expect the result to be `anim 0.25s 123` but instead it's `anim 0.25s [Object object]`.

**Comment by kof** [comment]
can you make a reproducible codesandbox example?

**Comment by kof** [comment]
Oh wait, I think its because of jss-expand, try regular css syntax

**Comment by brianmhunt** [comment]
It is indeed peculiar to jss-expand; it works with regular syntax.

**Comment by kof** [comment]
I am not sure if we need to fix this or just add a note in the docs

**Comment by kof** [comment]
Yeah, I was trying to avoid the isObservable detection within plugins and handle it within observable plugin completely, but you found a way where observable plugin can't detect it :)

**Comment by kof** [comment]
So the architectural idea is that observable jss plugin takes out the observable instances before everyone else, subscribes to them and other plugins shouldn't have to know anything about it.

**Comment by brianmhunt** [comment]
That might indeed be tricky :)

**Comment by kof** [comment]
Actually the observable plugin could go deeper into the tree and handle it, need to see if that would be too hard
